azure/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,12 +1,20 @@
 # Self-hosted Kubernetes assets (kubeconfig, manifests)
 module "bootkube" {
-  source = "git::https://github.com/kinvolk/terraform-render-bootkube.git?ref=d07243a9e7f6084cfe08b708731a79c26146badb"
+  source = "github.com/kinvolk/terraform-render-bootkube?ref=f5e41c83ec01e00187cb5c50cb7e9ec8275bd43e"
+
+  cluster_name = "${var.cluster_name}"
+  api_servers  = ["${format("%s.%s", var.cluster_name, var.dns_zone)}"]
+  etcd_servers = ["${formatlist("%s.%s", azurerm_dns_a_record.etcds.*.name, var.dns_zone)}"]
+  asset_dir    = "${var.asset_dir}"
+
+  networking = "${var.networking}"
+
+  # only effective with Calico networking
+  network_encapsulation = "vxlan"
+
+  # we should be able to use 1450 MTU, but in practice, 1410 was needed
+  network_mtu = "1410"
 
-  cluster_name          = "${var.cluster_name}"
-  api_servers           = ["${format("%s.%s", var.cluster_name, var.dns_zone)}"]
-  etcd_servers          = ["${formatlist("%s.%s", azurerm_dns_a_record.etcds.*.name, var.dns_zone)}"]
-  asset_dir             = "${var.asset_dir}"
-  networking            = "flannel"
```

Comment:
Why change all these variables in the upgrade? Is this needed to upgrade or is this a different *logical* change?

Reply:
This PR imports all the changes we missed from Typhoon.

Reply:
I can document those commits in this PR, do you know of a better of putting this information out there?

Reply:
Doing that is certanly very useful! I didn't understand why there were so many changes :-D